plugins/com.google.cloud.tools.eclipse.ui.util/src/com/google/cloud/tools/eclipse/ui/util/MessageConsoleUtilities.java

```diff
@@ -95,11 +95,15 @@ public static MessageConsole getMessageConsole(String consoleName,
     return console;
   }
 
-  public static <C extends MessageConsole> C createConsole(String name, ConsoleFactory<C> factory) {
+  public static <C extends MessageConsole> C createConsole(String name, ConsoleFactory<C> factory,
+      boolean showConsole) {
     ConsolePlugin plugin = ConsolePlugin.getDefault();
     IConsoleManager manager = plugin.getConsoleManager();
     C console = factory.createConsole(name);
     manager.addConsoles(new IConsole[]{console});
+    if (showConsole) {
+      manager.showConsoleView(console);
```

Comment:
showConsole feels like a separate operation that perhaps doesn't belong in this method. Could we move this part back where it was?